pkg/operator/ceph/config/config.go

```diff
@@ -71,6 +75,9 @@ var (
 	// VarLogCephDir defines Ceph logging directory. It is made overwriteable only for unit tests where it
 	// may be needed to send data intended for /var/log/ceph to a temporary test dir.
 	VarLogCephDir = "/var/log/ceph"
+
+	// VarCrashCephDir defines Ceph crash reports directory.
```

Comment:
What happens if you have daemons from multiple rook-ceph clusters on the same nodes? Do we include a subpath that is specific to the cluster so we don't mix up the crashes from different clusters?

Reply:
Yes, `VarCrashCephDir` is only on the container side, on the host, we use the namespace as a separator.